Source/Diagnostics/Diagnostics.cpp

```diff
@@ -574,6 +579,15 @@ Diagnostics::ComputeAndPack ()
             // Check that the proper number of components of mf_avg were updated.
             AMREX_ALWAYS_ASSERT( icomp_dst == m_varnames.size() );
 
+            if (m_diag_type == DiagTypes::TimeAveraged) {
+
+                amrex::Real real_a = 1.0;
```

Comment:
This should fix the clang-tidy error "variable 'real_a' of type 'amrex::Real' (aka 'float') can be declared 'const' [misc-const-correctness,-warnings-as-errors]":
```suggestion
                const amrex::Real real_a = 1.0;
```